lib/openstudio/alfalfa/creator.rb

```diff
@@ -114,19 +113,23 @@ def find_template(template)
         return false
       end
 
-      def resolve_template(mapping)
+      def resolve_template_from_mapping(mapping)
         cls = mapping['openstudio_class']
         k = @metadata_type.downcase
-        t = mapping[k]['template']
-        if @current_repo.has_term? @current_vocab[t]
+        template = mapping[k]['template']
```

Comment:
cls is unused and you can just put `@metadata_type.downcase` directly into the statement below